### PR TITLE
Replace PythX entry, add new CLI and Brownie

### PR DESCRIPTION
**Pull request by dmuhs**
This PR will
- add a deprecation notice for Armlet in favour of mythxjs
- remove the deprecated Shard Golang library
- move mythxjs from the tools to the libraries section
- refresh the PythX docs entry
- add a new entry for the `mythx-cli` tool
- add an entry for Brownie

Please note that this PR should only be merged once Brownie has actually approved the MythX integration PR: https://github.com/iamdefinitelyahuman/brownie/pull/218

### CODE REVIEW

File: docs/source/tools/brownie/index.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  contracts on the main-net, or for quick testing in a local environment.
+* **Deployment**: Automate the deployment of many contracts onto the
+  blockchain, and any transactions needed to initialize or integrate the
+  contracts.
+
+
+Scanning for Security Vulnerabilities
+-------------------------------------
+
+To prevent vulnerabilities from being introduced to the code base, Brownie a
```
Comment (b-mueller) [inline docs/source/tools/brownie/index.rst]: Brownie *includes* a plugin

File: docs/source/tools/brownie/index.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+plugin that integrates automated security scans using the MythX analysis API.
+Simply run :code:`brownie analyze` on your compiled project directory. This
+will send the compiled build artifacts to MythX for analysis. By default no
+login is required and the analysis is going to be executed as a trial user.
+To access more vulerability information, register for free on the MythX
```
Comment (b-mueller) [inline docs/source/tools/brownie/index.rst]: - missing "n" in vulnerability
"to obtain a complete vulnerability analysis"

File: docs/source/tools/brownie/index.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+website and pass your login data via environment variables or command
```
Comment (b-mueller) [inline docs/source/tools/brownie/index.rst]: command options -> command line arguments?

File: docs/source/tools/brownie/index.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      --gui                   Launch the Brownie GUI after analysis
+      --full                  Perform a full scan (MythX Pro required)
+      --interval=<sec>        Result polling interval in seconds [default: 3]
+      --async                 Do not poll for results, print job IDs and exit
+      --access-token          The JWT access token from the MythX dashboard
+      --eth-address           The address of your MythX account
+      --password              The password of your MythX account
+      --help -h               Display this message
+
+    Use analyze to submit your project to the MythX API for smart contract
```
Comment (b-mueller) [inline docs/source/tools/brownie/index.rst]: use the "analyze" command

**Comment by iamdefinitelyahuman** [comment]
https://github.com/iamdefinitelyahuman/brownie/pull/218 has merged :+1:

**Comment by dmuhs** [comment]
I have also fixed the same mistakes in the actual Brownie plugin documentation :+1:

**Pull request merged**